_maps/map_files/generic/CentCom.dmm

```diff
@@ -16736,6 +16715,10 @@
 	smooth = 1
 	},
 /area/centcom/holding)
+"Nf" = (
+/obj/machinery/autoylathe,
```

Comment:
Dont forget to add in plastic or a chem dispender to make it.
Also can use a new stack of glass/metal/wood

Reply:
Otherwise make a preloaded subtype, whatever works.